openmc/source.py

```diff
@@ -70,9 +73,19 @@ class Source:
 
     """
 
-    def __init__(self, space=None, angle=None, energy=None, time=None, filename=None,
-                 library=None, parameters=None, strength=1.0, particle='neutron',
-                 domains=None):
+    def __init__(
+        self,
+        space: Optional[openmc.stats.Spatial] = None,
+        angle: Optional[openmc.stats.Spatial] = None,
+        energy: Optional[openmc.stats.Univariate] = None,
+        time: Optional[openmc.stats.Univariate] = None,
+        filename: Optional[str] = None,
+        library: Optional[str] = None,
+        parameters: Optional[str] = None,
+        strength: float = 1.0,
+        particle: str = 'neutron',
+        domains: Optional[Union[openmc.Cell, openmc.Material, 'openmc.Universe']] = None
```

Comment:
The following change would allow you to remove the quotes:
```diff
diff --git a/openmc/__init__.py b/openmc/__init__.py
index f9abe5dc1e..cbf4d9ae2c 100644
--- a/openmc/__init__.py
+++ b/openmc/__init__.py
@@ -10,11 +10,11 @@ from openmc.material import *
 from openmc.plots import *
 from openmc.region import *
 from openmc.volume import *
-from openmc.source import *
 from openmc.weight_windows import *
-from openmc.settings import *
 from openmc.surface import *
 from openmc.universe import *
+from openmc.source import *
+from openmc.settings import *
 from openmc.lattice import *
 from openmc.filter import *
 from openmc.filter_expansion import *

```

Reply:
Thanks for this nice reordering to solve the circular imports, I've added it as requested